vformer/encoder/embedding/cvt.py

```diff
@@ -0,0 +1,91 @@
+import torch
+import torch.nn as nn
+
+
+class CVTEmbedding(nn.Module):
+    """
```

Comment:
Could you please a one-line description about the module here before listing the parameters?